datalake/src/main/java/com/sequenceiq/datalake/service/sdx/DistroxService.java

```diff
@@ -33,6 +34,10 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DistroxService.class);
 
+    private static final String STOP = "stop";
+
+    private static final String START = "start";
```

Comment:
**suggestion:** Consider an enum, rather than Strings:
```diff
diff --git a/datalake/src/main/java/com/sequenceiq/datalake/service/sdx/DistroxService.java b/datalake/src/main/java/com/sequenceiq/datalake/service/sdx/DistroxService.java
index 595394de75..49a332f3f1 100644
--- a/datalake/src/main/java/com/sequenceiq/datalake/service/sdx/DistroxService.java
+++ b/datalake/src/main/java/com/sequenceiq/datalake/service/sdx/DistroxService.java
@@ -34,9 +34,10 @@ public class DistroxService {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DistroxService.class);
 
-    private static final String STOP = "stop";
-
-    private static final String START = "start";
+    private enum Type {
+        STOP,
+        START;
+    }
 
     @Value("${dl.dh.polling.attempt:90}")
     private Integer attempt;
@@ -66,7 +67,7 @@ public class DistroxService {
             Polling.stopAfterAttempt(attempt)
                     .stopIfException(true)
                     .waitPeriodly(sleeptime, TimeUnit.SECONDS)
-                    .run(checkDistroxStatus(pollingCrn, STOP, this::stackAndClusterStopped));
+                    .run(checkDistroxStatus(pollingCrn, Type.STOP, this::stackAndClusterStopped));
         }
 
     }
@@ -79,12 +80,12 @@ public class DistroxService {
             Polling.stopAfterAttempt(attempt)
                     .stopIfException(true)
                     .waitPeriodly(sleeptime, TimeUnit.SECONDS)
-                    .run(checkDistroxStatus(pollingCrn, START, this::stackAndClusterStarted));
+                    .run(checkDistroxStatus(pollingCrn, Type.START, this::stackAndClusterStarted));
 
         }
     }
 
-    private AttemptMaker<Void> checkDistroxStatus(ArrayList<String> pollingCrn, String type, Function<StackV4Response, Boolean> statusChecker) {
+    private AttemptMaker<Void> checkDistroxStatus(ArrayList<String> pollingCrn, Type type, Function<StackV4Response, Boolean> statusChecker) {
         return () -> {
             List<String> remaining = new ArrayList<>();
             List<AttemptResult<Void>> results = pollingCrn.stream()
@@ -115,7 +116,7 @@ public class DistroxService {
         return AttemptResults.finishWith(null);
     }
 
-    private AttemptResult<Void> checkStatus(StackV4Response stack, String type) {
+    private AttemptResult<Void> checkStatus(StackV4Response stack, Type type) {
         ClusterV4Response cluster = stack.getCluster();
         if (Status.STOP_FAILED.equals(stack.getStatus()) || Status.START_FAILED.equals(stack.getStatus())) {
             LOGGER.info("Datahub stack " + type +
```